tests/test_stream.py

```diff
@@ -173,6 +173,18 @@ def testHNRLOG(
                 self.assertEqual(str(parsed), EXPECTED_RESULTS[i])

```

Comment:
You've ticked the checkbox in the PR template, but did you actually run your changes through the entire unittest suite? I'm seeing quite a few failures in /tests/test_stream.py, and /tests/test_socket.py hangs completely.

<img width="358" alt="Screenshot 2023-05-10 at 19 29 03" src="https://github.com/semuconsulting/pyubx2/assets/28569967/b34ba1fe-ed08-4fcf-8c50-fb622d09270d">